app/assets/javascripts/auth/cover.js

```diff
@@ -1,8 +0,0 @@
-jQuery(function ($) {
-  var rndNum;
-
-  if ($('body section.sign-up, body section.login').length) {
-    rndNum = Math.floor((Math.random() * 2) + 1);
-    $('body').addClass('massive-background-' + rndNum).hide().fadeIn(1000);
```

Comment:
Any idea what this `massive-background-n` is/was used for? I didn't find anything.